### PR TITLE
VIDLA-3480 - Added logic to allow publishers to defined Ad Marker CSS styles

### PR DESCRIPTION
**Pull request by graphicdeveloper**
These changes give publishers a way to style ad markers  on the options config object via options.adMarkers.markerStyle

### CODE REVIEW

**Comment by vpitts** [comment]
@graphicdeveloper - Can you send me a write up describing the new adMarkStyle option so that I can see what the option would look like in the doc.?

**Comment by graphicdeveloper** [comment]
As applied to most of the samples in the Brightcove Prebid Plugin docs, it would simply be adding an extra object to the options object, like so (I'll email you a full sample as well):
`options.adMarkerStyle = {
        'width': '5px',
        'border-radius': '10%',
        'background-color': 'white'
    }  `

File: src/MarkersHandler.js [hunk 1]
```diff
@@ -90,12 +91,16 @@ var markersHandler = function (vjs) {
 	        markersMap = {},
 	        markersList = [],
 	        // list of markers sorted by time
-	    		currentMarkerIndex = NULL_INDEX,
+			currentMarkerIndex = NULL_INDEX,
 	        markerTip = null,
 	        breakOverlay = null,
 	        overlayIndex = NULL_INDEX;
 
-	    function sortMarkersList() {
```
Comment (vpitts) [inline src/MarkersHandler.js]: It is a little unclear how the publisher-specific options are being applied to the markers - maybe we need some comment here?

Comment (graphicdeveloper) [reply src/MarkersHandler.js]: I added a comment to clarify. Please see the new commit below this comment.

**Comment by graphicdeveloper** [comment]
Both @vpitts and @ynisman have approved this PR - I'm merging it into the working dev branch - VIDLA-VER-0.2.1+

**Pull request merged**